src/pages/api/image/[url].ts

```diff
@@ -11,18 +11,40 @@ export default async function handler(
 
   const imageUrl = req.query?.url as string
   if (!imageUrl) {
```

Comment:
@filipviz this LGTM. one nice improvement we could make would be to cache responses for this endpoint. Given we know the URI will always point to the same file (otherwise, it would have a different CID), the cache duration could be pretty long.